authz/providers/azure/rbac/checkaccessreqhelper.go

```diff
@@ -226,53 +227,52 @@ func getResultCacheKey(subRevReq *authzv1beta1.SubjectAccessReviewSpec) string {
 	return cacheKey
 }
 
-func prepareCheckAccessRequestBody(req *authzv1beta1.SubjectAccessReviewSpec, clusterType, resourceId string, retrieveGroupMemberships bool) (*CheckAccessRequest, error) {
+func prepareCheckAccessRequestBody(req *authzv1beta1.SubjectAccessReviewSpec, clusterType, resourceId string) (*CheckAccessRequest, error) {
 	/* This is how sample SubjectAccessReview request will look like
-	{
-		"kind": "SubjectAccessReview",
-	    	"apiVersion": "authorization.k8s.io/v1beta1",
-	    	"metadata": {
-	        	"creationTimestamp": null
-	    	},
-	    	"spec": {
-	        	"resourceAttributes": {
-	            		"namespace": "default",
-		            	"verb": "get",
-				"group": "extensions",
-				"version": "v1beta1",
-				"resource": "deployments",
-				"name": "obo-deploy"
-	        	},
-			"user": "user@contoso.com",
-			"extra": {
-				"oid": [
-	    			"62103f2e-051d-48cc-af47-b1ff3deec630"
-			]
-	        	}
-	    	},
-	    	"status": {
-	        	"allowed": false
-	    	}
-	}
+		{
```

Comment:
can we clean up these tabs?

Reply:
they are added by the make gen fmt command needed for the linter